_config.yml

```diff
@@ -5,7 +5,7 @@ email: contact@codefornepal.org
 description: >-
   Community of volunteers helping address Nepal's challenges using civic tech
   to innovate, organize and advocate digital literacy
-baseurl: "/website20" # the subpath of your site, e.g. /blog
+baseurl: "" # the subpath of your site, e.g. /blog
```

Comment:
Thanks for these changes! should make it easier to modify. 

I believe we still need to keep this to `/website20` since the repo name isn't changed to `code4nepal.github.io` yet